### ISSUE TITLE
supabase starts edge function runtime automatically but doesn't support --env-file

### ISSUE THREAD

**Issue by evelant** [supabase/cli#1001]
# Bug report

- [X ] I confirm this is a bug with Supabase, not with my own application. 
- [ X] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

I need to use a custom env file when running my functions locally due to the need for DENO_AUTH_TOKENS to load code from a private github repo (to work around the inability to reference local code outside of the functions dir). 

This works fine with `supabase functions serve --env-file supabase/.env.local`.

Now that the cli automatically starts the edge runtime it is not possible to specify the env file which results in my local edge functions failing. 

```
supabase start --env-file supabase/.env.local
Error: unknown flag: --env-file
```

## To Reproduce

1. supabase start --env-file your_path_to/.env
2. Observe error

## Expected behavior

There should be the option to load a .env file for edge functions when auto-started with `supabase start`

## System information

- OS: macOS
- Browser (if applies) N/A
- Version of supabase: 1.49.4
- Version of Node.js: 16.19.0

**Comment by sweatybridge** [comment]
Hello, we added a default location for loading env file from, ie. `supabase/functions/.env`

Is that sufficient for your use case?

**Comment by evelant** [comment]
Yep that works fine! Perhaps the --help for `supabase start` should mention it however.
